pkg/util/pki/common.go

```diff
@@ -243,3 +245,12 @@ func EnsureControllerReference(ctx context.Context, user *v1alpha1.KafkaUser,
 	}
 	return nil
 }
+
+// TruncatedCommonName ensures that the passed-in common name doesn't exceed the specified number of characters
+func TruncatedCommonName(name string, maxLen int) string {
```

Comment:
We could use `TruncateCommonName` instead so it's a verb as a function name?
Maybe we could move this to a common/util package and make it a generic string truncate function.